src/nIsForName.js

```diff
@@ -22,7 +22,11 @@
  * @return {string[]}
  */
 function nIsForName(names) {
-  // write code here
+  return names.filter((name) => {
+    return name.split(' ').some(function(arrName) {
+      return arrName[0] === 'N';
```

Comment:
startsWith

Reply:
this is a string method, not a variable name